### PR TITLE
Enable security scan  & Fix fortify and dependency stages in nightly

### PR DESCRIPTION
**Pull request by mounikahmcts**
The PR fixes
1. Fortify scan in nightly job
2. Security scan in nightly job
3. Migrating helm to latest version
4. Enabling path to live functional tests in nightly build

### CODE REVIEW

File: src/integration-test/tests/citizen/claim/helpWithFees_test.ts [hunk 1]
```diff
@@ -9,11 +9,6 @@ Scenario('Navigate Payment Page by providing claim details... @smoke-test', { re
   await claimSteps.makeAClaimAndNavigateUpToPayment(I)
 })
 
-// The @citizen-smoke-test tag used for running smoke tests with pre-registered user for help with fees
```
Comment (mounikahmcts) [inline src/integration-test/tests/citizen/claim/helpWithFees_test.ts:12]: Removing this test as the same steps are executed in 
Scenario('Submit Claim via HWF Reference... @citizen', { retries: 3 }, async (I: I) => {
  await claimSteps.makeAHwfClaimAndSubmit(I)
})

File: src/integration-test/tests/citizen/claim/steps/claim.ts [hunk 2]
```diff
@@ -339,67 +339,6 @@ export class ClaimSteps {
     this.enterClaimEvidence()
   }
 
-  async makeAHwfClaimAndNavigateUpToPayment (I: I) {
```
Comment (mounikahmcts) [inline src/integration-test/tests/citizen/claim/steps/claim.ts:342]: Removing this test as the same steps are executed in 
Scenario('Submit Claim via HWF Reference... @citizen', { retries: 3 }, async (I: I) => {
  await claimSteps.makeAHwfClaimAndSubmit(I)
})

File: charts/cmc-citizen-frontend/Chart.yaml [hunk 3]
```diff
@@ -12,7 +12,7 @@ maintainers:
     email: cmc-developers@HMCTS.NET
 dependencies:
   - name: nodejs
-    version: 2.4.0
+    version: 2.4.2
```
Comment (mounikahmcts) [inline charts/cmc-citizen-frontend/Chart.yaml:15]: Helm fix

File: Dockerfile [hunk 4]
```diff
@@ -1,5 +1,8 @@
 # ---- Base image ----
 FROM hmctspublic.azurecr.io/base/node:12-alpine as base
+
+USER hmcts
```
Comment (mounikahmcts) [inline Dockerfile:4]: Fortify fix

**Review by n-clarke-777 — APPROVED** [review]

**Pull request merged**